frontend/src/lib/components/CommandPalette/commandPaletteLogic.tsx

```diff
@@ -510,9 +510,9 @@ export const commandPaletteLogic = kea<commandPaletteLogicType>([
                     },
                     {
                         icon: IconLive,
-                        display: 'Go to Events explorer',
+                        display: 'Go to Activity',
```

Comment:
Same here with the feature flag. This should be changed for everyone.

Reply:
Might be being thick – what would you change it to?